packages/golden_toolkit/lib/src/golden_builder.dart

```diff
@@ -192,6 +192,7 @@ Future<void> screenMatchesGolden(
   String goldenFileName, {
   Finder finder,
   CustomPump customPump = _onlyPumpAndSettle,
+  bool skip = false,
```

Comment:
should add method level documentation

```skip = true will skip the golden file assertion. This may be necessary if your development platform is not the same as your CI platform```

Reply:
Added!